src/constants.ts

```diff
@@ -124,3 +124,15 @@ export const URL_QUERY_KEYS = {
 export const URL_HASH_KEYS = {
   addLibrary: "addLibrary",
 } as const;
+
+export const CANVAS_ACTIONS = {
+  clearCanvas: true,
+  export: true,
+  loadScene: true,
+  saveAsScene: true,
+  saveScene: true,
```

Comment:
why not add in default props instead ?

Reply:
Won't default props be overridden if if the user passes something like this
```
UIOptions = {
  canvasActions: {
    export: false
  }
};
```

Reply:
Before passing it to `App.tsx` you can do something like 👇 in `excalidraw/index.tsx`
```
UIOptions = {
  canvasActions: {
    ...UIOPTIONS.DEFAULT_CANVAS_ACTIONS,
    ...this.props.UIOptions.canvasActions,
    }
}
where `UIOPTIONS.DEFAULT_CANVAS_ACTIONS` is the default value for canvas actions
This way you will always have correct state of the props.

Reply:
I'm already doing something like that. I'll set the default value as `UI_OPTIONS`.

```
UIOptions = {
  canvasActions: {
    ...CANVAS_ACTIONS,
    ...(props.UIOptions?.canvasActions || {}),
  },
};
```

Instead of naming it `DEFAULT_CANVAS_ACTIONS` inside `UI_OPTIONS`, I'll keep the name just `canvasActions` so that we can set `UI_OPTIONS` as the default prop.

Reply:
I see, In that case I would suggest not to have this variable at all and just have a variable and use it for spreading so as to make it clear that these are default options.
```
DEFAULT_UI_OPTIONS = {
  canvasActions: {
    clearCanvas: true,
     export: true,
     loadScene: true,
     saveAsScene: true,
     saveScene: true,
  }
}